docs/modules/ROOT/pages/gsn_advanced.adoc

```diff
@@ -0,0 +1,238 @@
+= Advanced GSN Guide (Bouncers)
+
+This guide shows you different strategies (Bouncers) to accept transactions via Gas Station Network.
+
+First, we will explain the Bouncer concept and then we will showcase how to use the two most common strategies.
+
+Finally, we will cover how to create your own custom Bouncer.
+
+If you're still learning about the basics of the Gas Station Network, you should head over to our xref:api:gsn.adoc[GSN Guide], which will help you get started from scratch.
+
+== GSN Bouncers
+
+A *GSN Bouncer* decides which transaction get approved and which transaction get rejected. Bouncers are a key concept within GSN. Dapps need Bouncers to prevent malicious users to spend the transactions subsidy.
```

Comment:
```suggestion
A *GSN Bouncer* decides which transaction gets approved and which transaction gets rejected. Bouncers are a key concept within GSN. Dapps need Bouncers to prevent malicious users to spend the transactions subsidy.
```